compute_tools/src/spec.rs

```diff
@@ -679,6 +679,14 @@ pub fn handle_extensions(spec: &ComputeSpec, client: &mut Client) -> Result<()>
             info!("creating system extensions with query: {}", query);
             client.simple_query(query)?;
         }
+
+        if libs.contains("anon") {
```

Comment:
why do you need to create it?

Reply:
I'm not sure that it's necessary

Reply:
I don't get your question..
Why do we need to create extension? Because users cannot create it, because they don't have superuser access.

Reply:
but how they'll be able to use it? 

```
pg_catalog is always owned by the initial superuser role.
So in practice, you need to be superuser to run the SECURITY LABEL command, 
which means that you need to be superuser to install the extension.
```

even if we create the extension on our own

Reply:
do we need to allow run `SECURITY LABEL` for `neon_superuser` members?

Reply:
anon (by default) allows to use non-trusted schemas. so this security label doesn't really matter

Though, I bumped into some other superuser checks.

This set of commands (run from superuser) enables [static masking functions](https://postgresql-anonymizer.readthedocs.io/en/latest/static_masking/)
```
CREATE EXTENSION IF NOT EXISTS anon
SELECT anon.init();
 // probably GRANT ALL ON SCHEMA anon TO neon_superuser WITH GRANT OPTION
 // will be more secure here
GRANT SELECT ON ALL SEQUENCES IN SCHEMA anon TO PUBLIC;
```

but for dynamic masking there are some more checks in anon functions.
I'll try to rewrite them to check for `neon_superuser` instead.